.tx/config

```diff
@@ -9,4 +9,5 @@ trans.fr = locale/fra/LC_MESSAGES/django.po
 trans.es = locale/es/LC_MESSAGES/django.po
 trans.hi = locale/hin/LC_MESSAGES/django.po
 trans.pt = locale/por/LC_MESSAGES/django.po
+trans.sw = locale/sw/LC_MESSAGES/django.po
```

Comment:
@czue I'm not sure that is all that should be done to have translations in transifex. Could you ping someone responsible for transifex, please?